train/tictactoe.py

```diff
@@ -21,19 +21,27 @@ class TicTacToe(TrainableGame):
     PLANES_NUM = 3
     MOVE_NUM = BOARD_SIZE * BOARD_SIZE
 
+    """
+    planes: [u64; 3]
+    probs: [f32; 9]
+    winner: i8
+    """
+    ENTRY_FORMAT = struct.Struct("24s36sb")
```

Comment:
Can you generalize the numbers here? 
Consider using Construct (library) instead of Struct. It's a arguably a bit of an overkill but makes the code more readable